test/examples/ucp_hello_world.c

```diff
@@ -63,6 +64,7 @@ enum ucp_test_mode_t {
 } ucp_test_mode = TEST_MODE_PROBE;
 
 static uint16_t server_port = 13337;
+static long test_string_length = 16;
```

Comment:
the test string is defined globally - hence the default size for the test string should be defined globally.